src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,68 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final int NUMBER_ZERO = 0;
```

Comment:
```suggestion
    private static final int MIN = 0;
```
값의 의미를 나타내줄 수 있는 네이밍은 어떨까요?  😄

Reply:
MIN이 훨씬 더 좋은 것 같습니다. 👍 
말씀해주신 부분 고려하여 네이밍에 더 신경쓰도록 하겠습니다.